### PR TITLE
Setup navigation router

### PR DESCRIPTION
**Pull request by fareed-ah**

### CODE REVIEW

**Review by zachary-nguyen — APPROVED** [review]
LGTM!

**Review by MaximeNdutiye — APPROVED** [review]

**Comment by MaximeNdutiye** [comment]
tophat looks good 👍🏿

File: src/main/js/views/ErrorPage.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const ErrorPage = () => {
+    return (
+      <div>
+       Rip
```
Comment (ericcosoreanu) [inline src/main/js/views/ErrorPage.js:6]: Please add a more descriptive error message.

Comment (MaximeNdutiye) [reply src/main/js/views/ErrorPage.js:6]: good catch Eric!

Comment (fareed-ah) [reply src/main/js/views/ErrorPage.js:6]: Wow didn't notice that! Thanks for letting me know.

**Review by ericcosoreanu — APPROVED** [review]

**Pull request merged**